4week/src/hooks/mutations/useDeleteLike.ts

```
@@ -0,0 +1,61 @@
+
+            return {previousLpPost, newLpPost}
+        },
+
+        onError: (err,newLp,context) => {
+            console.log(err,newLp);
+            queryClient.setQueryData([QUERY_KEY.lps,newLp.lpid], context?.previousLpPost?.data.id)
+        },
+
+        // 서버 상태 동기화
+        // onSetteled는 API 요청이 끝난 후 성공하든 실패하든 실행
+        onSettled : async(data, error, variables, context) => {
```

ESLint

[error] 'context' is defined but never used.

(@typescript-eslint/no-unused-vars)